src/_components/AddAsset/AddAssetContainer.jsx

```diff
@@ -0,0 +1,173 @@
+import _ from 'lodash';
+import PropTypes from 'prop-types';
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { ToastMessage } from '../../_utils/ToastMessage';
+
+import AddAssetComponent from '../../components/AddAsset/AddAssetComponent';
+
+import { loadCategories } from '../../_actions/category.actions';
+import { loadSubCategories } from '../../_actions/subcategory.actions';
+import { loadAssetTypes } from '../../_actions/assetTypes.actions';
+import { loadAssetMakes } from '../../_actions/assetMakes.actions';
+import { loadModelNumbers } from '../../_actions/modelNumbers.actions';
+import { createAsset } from '../../_actions/asset.actions';
+
+import {
+  filterSubCategories,
+  filterAssetTypes,
+  filterAssetMakes,
+  filterModelNumbers
+} from '../../_utils/filterDropdowns';
+
+class AddAssetContainer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      filteredSubCategories: [],
+      filteredAssetTypes: [],
+      filteredAssetMakes: [],
+      filteredModelNumbers: [],
+      modelNumber: 0,
+      serialNumber: '',
+      assetTag: '',
+      assets: typeof(props.assets) !== "undefined" ? props.assets: []
```

Comment:
No need for the `typeof` to check if assets is undefined. This should be:
```
assets: props.assets || []
```

You can infer from the screenshot that `[]` will be used when the first value is falsy/undefined as demonstrated by `getVal` variable:

<img width="406" alt="screen shot 2018-06-08 at 8 00 01 am" src="https://user-images.githubusercontent.com/12892171/41139718-1a629aba-6af2-11e8-9513-07094af5a2cb.png">

Reply:
I see your point. I have made the adjustment.